gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -962,7 +962,7 @@ public void putCourseGradesInMatrix(Map<String, GbStudentGradeInfo> matrix, List
 
 		// Setup the course grade formatter
 		// TODO we want the override except in certain cases. Can we hard code this?
-		final CourseGradeFormatter courseGradeFormatter = new CourseGradeFormatter(gradebook, role, isCourseGradeVisible, settings.getShowPoints(), false);
+		final CourseGradeFormatter courseGradeFormatter = new CourseGradeFormatter(gradebook, role, isCourseGradeVisible, settings.getShowPoints(), true);
```

Comment:
why was this ever false? It means to use the course grade override when showing the data.

Reply:
Looks like a regression from SAK-33849 #5245

Reply:
@bjones86 was there a reason for the change from true to false in the above PR or is this a regression?

Reply:
Looks like this change was accidental.